### DISCUSSION TITLE
Add items to ListBox DataGrid.. in code behind

### DISCUSSION THREAD

**Discussion by ride62** [AvaloniaUI/Avalonia#8969] [Q&A] [answered]
there is  something equivalent to wpf "ListBox.Items.Add"

**Comment by maxkatz6** [comment, marked as answer]
ListBox.Items = new List<T>()
And you can cast it to the list type later if you want to add on the flight.

**Reply by maxkatz6** [reply]
And I recommend to use ObservableCollection or AvaloniaList as a list type, so changes would be dynamically updated

**Reply by ride62** [reply]
thank you
